ksql-engine/src/main/java/io/confluent/ksql/physical/PhysicalPlanBuilder.java

```diff
@@ -214,12 +220,23 @@ private String addTimeSuffix(String original) {
     return String.format("%s_%d", original, System.currentTimeMillis());
   }
 
-  private KafkaStreams buildStreams(
-      final StreamsBuilder builder,
-      final String applicationId,
-      final KsqlConfig ksqlConfig,
-      final Map<String, Object> overriddenProperties
-  ) {
+  private void updateListProperty(Map<String, Object> properties, String key, Object value) {
+    Object obj = properties.getOrDefault(key, new java.util.LinkedList<String>());
+    List valueList;
+    if (obj instanceof String) {
+      String asString = (String)obj;
+      valueList = new LinkedList<>(Arrays.asList(asString));
```

Comment:
why `Arrays.asList(...)` when you have a string?

Reply:
The setting accepts a comma-separated list, and there was a .split("\\s*,\\s*") that got dropped somehow when I was moving some code around. Fixing.